dbbackup/management/commands/mediabackup.py

```diff
@@ -113,7 +137,7 @@ def get_backup_file_list(self):
         if server_name:
             server_name = '-%s' % server_name
 
-        media_re = re.compile(r'^%s%s-(.*)\.media\.tar\.gz' % (self.get_databasename(), server_name))
+        media_re = re.compile(r'^%s%s-(.*)\.media\.tar' % (self.get_databasename(), server_name))
```

Comment:
Here we get a list of existing files.
I think we should get a full list of files (compressed or not) does not matter what option we use right now, because with --clean option we should clean all old backup files even their compression type is different.
We should match a files with '.gz' extension and without it. That is why I removed '.gz' from regexp pattern.